### PR TITLE
Add Hardening Role Documentation

### PR DESCRIPTION
**Pull request by MacHu-GWU**
## Why

We need the README.md for this rhel8 hardening role

## How

I use https://github.com/GSA/ise-hardening-template as the base doc template, and borrowed lots of snippet from https://github.com/GSA/ansible-os-ubuntu18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,157 +1,56 @@
-# cis_security
+# Redhat 8 CIS Benchmark
```
Comment (kannan-gsa) [inline README.md:1]: Please replace all the 'CIS' with 'GSA'

File: README.md [hunk 2]
```diff
@@ -1,157 +1,56 @@
...
 
-A role to implement Center for Internet Security (CIS) controls for RHEL (7-8) and RHEL clones (Oracle, CentOS), SLES 15, and Ubuntu 18.04 LTS and certain Windows servers.
+A role to implement Center for Internet Security (CIS) controls for RHEL 8. It is based on [cis-security](https://github.com/dsglaser/cis-security)
```
Comment (kannan-gsa) [inline README.md:3]: Please replace this line with the following:
This role configures Red Hat Enterprise Linux (RHEL) 8.0 machine to be GSA compliant. Level 1 and 2 profiles will be applied by default based on [RHEL 8.0 GSA Benchmarks](https://docs.google.com/spreadsheets/d/14q7seglcBihKqs6muSQlCllVvzDmVOjS7PY758iB0aM)

File: README.md (hunk 1 above)
Comment (MacHu-GWU) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (MacHu-GWU) [reply README.md]: fixed

**Pull request merged**